caption.go

```diff
@@ -47,19 +47,19 @@ func (c Caption) FormatEnd() string {
 	return formatTimestamp(c.End, ".")
 }
 
-func formatTimestamp(timestamp float64, sep string) string {
-	value := int(timestamp / 1000)
+func formatTimestamp(timestamp *float64, sep string) string {
```

Comment:
is there any way we can leverage the go time package in this function?

Reply:
Do you mean instead of using pointer here?

Reply:
ah, I mean to parse/format the float64 timestamp, it looks to be done manually in the function

Reply:
I can create a separate PR for just this.